pkgs/top-level/all-packages.nix

```diff
@@ -13927,6 +13927,8 @@ with pkgs;
 
   torrent7z = callPackage ../tools/archivers/torrent7z { };
 
+  trrntzip = callPackage ../tools/archivers/trrntzip { };
```

Comment:
I think the package should be moved to `pkgs/by-name/tr/trrntzip/package.nix`

Reply:
At the time I wrote this package the `pkgs/by-name` hierarchy didn't exist. But now that I'm modifying the package anyway, why not